src/app/tokens/components/TokenTableCell/TokenActionCell.tsx

```diff
@@ -0,0 +1,100 @@
+import { BiDotsVerticalRounded, BiImport, BiTransfer } from "react-icons/bi";
+import {
+  Button,
+  ButtonProps,
+  Dropdown,
+  DropdownItem,
+  DropdownMenu,
+  DropdownTrigger,
+  Tooltip,
+} from "@nextui-org/react";
+import React, { useMemo } from "react";
+
+interface TokenActionCellProps {
+  isDisabled?: boolean;
+  isLoading?: boolean;
+  onTransferClick: () => void;
+  onImportClick: () => void;
+}
+
+interface ButtonItem extends Record<string, any> {
+  onClick: () => void;
+  color: ButtonProps["color"];
+  key: string;
+  label: React.ReactNode;
+  icon: React.ReactNode;
+}
+
+export const TokenActionCell = ({
+  isDisabled,
+  isLoading,
```

Comment:
It is okay for isLoading to not have a default value, but let's set a default value for consistency.